src/openrct2/object/LargeSceneryObject.cpp

```diff
@@ -15,10 +15,8 @@
 #pragma endregion
 
 #include "../core/IStream.hpp"
-#include "../core/Memory.hpp"
```

Comment:
This include is removed in many places, but it's still in use, just a few lines down in `LargeSceneryObject::~LargeSceneryObject()`. Please only remove when file does not make use of any of `Memory::`